tests/test_cli.py

```diff
@@ -183,7 +183,7 @@ def test_sync_command_integration(dependencies, clean_test_dir):
             assert all(
                 d.name.startswith("__") for d in TEST_SRC_VENDOR.iterdir() if d.is_dir()
             ), (
-                f"Vendor directory should be empty of packages but contains: {list(TEST_SRC_VENDOR.iterdir())}"
+                f"python_modules directory should be empty of packages but contains: {list(TEST_SRC_VENDOR.iterdir())}"
```

Comment:
```suggestion
                f"python_modules directory must be empty of packages but contains: {list(TEST_SRC_VENDOR.iterdir())}"
```
[RFC-2119](https://datatracker.ietf.org/doc/html/rfc2119)